doc/changes.rst

```diff
@@ -246,7 +246,12 @@ Changes to individual commands
   * Dropped ``--pkg`` option. Positional arguments can now be used to specify packages to check closure for.
 
 ``repolist``
-  * Options ``-v`` and ``--verbose`` have been removed. The functionality is replaced by the ``repoinfo`` command, which was already introduced in DNF4.
+  * The ``repolist`` and ``repoinfo`` commands are now subcommands of the ``repo`` command: ``repo list`` and ``repo info``.
+
+    * Original commands still exist as compatibility aliases.
+
+  * Options ``-v`` and ``--verbose`` have been removed. The functionality is replaced by the ``repo info`` command, which was already introduced in DNF4.
```

Comment:
> ``repo info`` command, which was already introduced in DNF4.

I don't think DNF4 has such a command.

Reply:
Right, I'll fix it. It was meant like it was already there with the `repoinfo` (now `repo info`).